src/lib.rs

```diff
@@ -1,3 +1,27 @@
+//! Clarity is a lightweight Ethereum client that's cross-compilation friendly
+//!
+//! # Getting started
+//! Here's an example of the lifetime of an Ethereum transaction made with Clarity:
+//! ```rust
+//! use clarity::{Address, Signature, Transaction, PrivateKey};
+//!
+//! let key_buf: [u8; 32] = rand::random();
+//! let key = PrivateKey::from_slice(&key_buf).unwrap();
```

Comment:
That would be interesting:

```rust
println!("private key = 0x{:x?}", key);
println!("public key = 0x{:x?}", key.to_public_key().unwrap());
```
At least showing public key would be interesting example.